plugins/parsers/graphite/parser_test.go

```diff
@@ -94,6 +94,13 @@ func TestTemplateApply(t *testing.T) {
 			measurement: "cpu.load",
 			tags:        map[string]string{"zone": "us-west"},
 		},
+		{
+			test:        "conjoined fields",
```

Comment:
here you should test that the correct fields get applied.

ie, this should test that measurement=="cpu.util" and field=="idle.percent" (feel free to make a separate test function for testing field templates)